tutorials/setup-minecraft/index.mdx

```diff
@@ -70,11 +70,11 @@ The Minecraft server is a Java application and runs perfectly on [Scaleway Eleme
 
 </Message>
 
-Deploying your own Minecraft server can be done in a few easy steps on a [Scaleway Elements Developent Instance](https://www.scaleway.com/en/virtual-instances/development/). In case you do not have an Instance yet, start by [deploying your first instance](/compute/instances/how-to/create-an-instance/).
+Deploying your own Minecraft server can be done in a few easy steps on a [Scaleway Elements Development Instance](https://www.scaleway.com/en/virtual-instances/development/). In case you do not have an Instance yet, start by [deploying your first Instance](/compute/instances/how-to/create-an-instance/).
```

Comment:
```suggestion
Deploying your own Minecraft server can be done in a few easy steps on a [Scaleway Development Instance](https://www.scaleway.com/en/virtual-instances/development/). In case you do not have an Instance yet, start by [deploying your first Instance](/compute/instances/how-to/create-an-instance/).
```